contrib/broker/k8s/Dockerfile

```diff
@@ -20,13 +20,16 @@ RUN export DEBIAN_FRONTEND=noninteractive \
  && apt-get update \
  && apt-get install -y \
     ca-certificates \
+    curl \
+ && curl -Ls -o /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+    http://storage.googleapis.com/kubernetes-helm/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+ && mkdir -p /opt/services \
+ && tar -C /opt/services -x -z -f /tmp/helm-${HELM_VERSION}-linux-amd64.tar.gz \
+    --strip-components=1 linux-amd64/helm \
+ # cleanup
```

Comment:
Ah... I'm sorry. I didn't understand that he meant the comment.

We (Deis) have used comments like this without any ill-effect.

@duglin can you elaborate on the "fix"? Are you saying that this was fine in older versions of Docker but not in a newer or upcoming version?